diff -ruN b/gen-py/atcd/Atcd-remote c/gen-py/atcd/Atcd-remote
--- b/gen-py/atcd/Atcd-remote	2013-09-04 00:46:22.454925102 +0000
+++ c/gen-py/atcd/Atcd-remote	2013-09-04 00:46:10.651829615 +0000
@@ -15,8 +15,10 @@
 from thrift.transport import THttpClient
 from thrift.protocol import TBinaryProtocol
 
-from atcd import Atcd
-from atcd.ttypes import *
+#from atcd import Atcd
+#from atcd.ttypes import *
+import Atcd
+from ttypes import *
 
 if len(sys.argv) <= 1 or sys.argv[1] == '--help':
   print ''
